aviary/subsystems/propulsion/test/test_propeller_performance.py

```diff
@@ -531,7 +532,7 @@ def test_tip_mach(self):
         assert_check_partials(partial_data, atol=1e-4, rtol=1e-4)
 
 
-class TipSpeedLimitTest(unittest.TestCase):
+class TipSpeedLimitest(unittest.TestCase):
```

Comment:
is this renaming intentional? It looks like it was an accident?

Reply:
I did a global replacement. Know it is named back.